R/formula-tools.R

```diff
@@ -294,18 +297,27 @@ rw <- function(time, by, type = c("independent", "dependent")) {
 #' epinowcast:::construct_rw(rw(week, day_of_week), data)
 construct_rw <- function(rw, data) {
   if (!inherits(rw, "enw_rw_term")) {
-    stop("rw must be a random walk term as constructed by rw")
+    rlang::abort(
+      paste(
```

Comment:
```suggestion
      paste0(
```
... also, any consideration of wrapper to give `stop(...)`s behavior?